internal/version/version.go

```diff
@@ -1,3 +1,3 @@
 package version
 
-const Version = "1.5.0-beta1"
+const Version = "1.5.0-beta2"
```

Comment:
1.5.0-beta2 is already released, but I missed updating the version string in that release.